frontend/pages/Landing.vue

```diff
@@ -5,12 +5,29 @@ import MeetupSpeakers from '@/components/Landing/MeetupSpeakers.vue'
 import AboutMeetup from '@/components/Landing/AboutMeetup.vue'
 import JoinUs from '@/components/Landing/JoinUs.vue'
 import Counters from '@/components/Landing/Counters.vue'
+import { onMounted, ref, computed } from 'vue'
+import { useFindNextMeetup } from '@/composables/useFindNextMeetup.js'
+
+const meetups = ref([])
+
+const findNextMeetup = computed(() => {
+  if (meetups.value.length == 0) return {}
+  const { nextMeetup } = useFindNextMeetup(meetups.value) 
+  return nextMeetup.value
+})
+
+onMounted(async() => {
+    await fetch('/api/meetups.json').then((response) => response.json()).then((data) => {
+    meetups.value = data
+  })
+})
+
 </script>
 
 <template>
   <hero-section/>
-  <meetup-indicator/>
-  <meetup-speakers/>
+  <meetup-indicator :next-meetup="findNextMeetup"/>
+  <meetup-speakers :next-meetup="findNextMeetup"/>
```

Comment:
Look, you are calling `findNextMeetup` twice here. Maybe it would be better to call it once in your `script` section and pass result here? Or maybe `MeetupIndicator` and `MeetupSpeakers` could handle different objects after all?

Reply:
Right. I have changed it. I called it now in script section, please take a look if it is optimized